spec/mailers/notifier_spec.rb

```diff
@@ -8,3 +8,45 @@
     expect(email.body).to include "service #{service_1.url}"
   end
 end
+
+describe 'ServiceChangeNotifier' do
+  include ServiceChangeNotifier
+  describe 'notify_services_of_changes' do
+    before(:each) do
+      @service_1 = create :service
+      @service_2 = create :service
+      @model_1 = create :model, service: @service_1
+    end
+
+    it 'Notifies all services with a list of all services/models' do
+      expected_params = [
+        { url: @service_1.url,
+          models: [{ name: @model_1.name }]
+        },
+        { url: @service_2.url,
+          models: []
+        }
+        ].to_json
+      expect(Net::HTTP).to receive(:post_form)
+        .with(URI(@service_1.url), 'services' => expected_params)
+        .and_return Net::HTTPResponse.new('1.0', '200', '')
```

Comment:
It'd be nice to document or comment what these arguments to `Net::HTTPResponse.new` are.